scripts/python/junglecatsoftware/awstoolkit/outputFormatters.py

```diff
@@ -5,18 +5,19 @@ def printBigHeader( author = 'Jungle Cat Software', package = 'AWS-Toolkit', fun
   firstLineLen = len( firstLine )
   secondLine = function
   secondLineLen = len( secondLine )
-
+  
```

Comment:
There really isn't a convention with that... whatever works.  :+1:   I think it was left over actually...